### PR TITLE
backend connect, error label activated etc

### PR DESCRIPTION
**Pull request by shayywun**
img_15d2b52dfe50-1

### CODE REVIEW

File: ChemoTracker/components/Login/Login.js [hunk 1]
```diff
@@ -7,15 +7,12 @@ import styles from '../../styles/login_screen.js';
...
-  capi = 'http://ec2-52-15-106-40.us-east-2.compute.amazonaws.com:8000';
-
-
-
   state = {
     fontLoaded: false,
     errorLabel: '',
     loginUsername: '',
-    loginPassword: '',
+		loginPassword: '',
```
Comment (lsivasothy) [inline ChemoTracker/components/Login/Login.js]: spacing looks weird

Comment (shayywun) [reply ChemoTracker/components/Login/Login.js]: lmao i didnt even notice that. fixing!

File: ChemoTracker/components/Login/Login.js [hunk 2]
```diff
@@ -7,15 +7,12 @@ import styles from '../../styles/login_screen.js';
 
 export default class Login extends React.Component {
 
-  capi = 'http://ec2-52-15-106-40.us-east-2.compute.amazonaws.com:8000';
-
-
-
   state = {
```
Comment (lsivasothy) [inline ChemoTracker/components/Login/Login.js]: add this inside a constructor (look at Calendar.js)

File: ChemoTracker/components/Login/Login.js [hunk 3]
```diff
@@ -74,18 +77,26 @@ export default class Login extends React.Component {
         <TextInput
           keyboardType="email-address"
           underlineColorAndroid="transparent"
+          onChangeText={(text)=>{this.setState({loginUsername:text})}}
+          onSubmitEditing={(event) => {
+									this.refs.PasswordField.focus();
```
Comment (lsivasothy) [inline ChemoTracker/components/Login/Login.js]: another spacing issue 😛

File: ChemoTracker/styles/login_screen.js [hunk 4]
```diff
@@ -62,6 +62,11 @@ const styles = StyleSheet.create({
   text:{
     color: 'rgb(251,91, 27)',
     fontWeight: '700',
+  },
+  errorLabel:{
+    fontSize: 14,
+		color: 'rgb(254, 40, 81)',
```
Comment (lsivasothy) [inline ChemoTracker/styles/login_screen.js]: another one 😛

**Review by lsivasothy — COMMENTED** [review]
Other than minor comments, PR looks great!

File: ChemoTracker/components/Login/Login.js (hunk 3 above)
Comment (shayywun) [reply ChemoTracker/components/Login/Login.js]: i think its github. i dont have this on my editor. i'll fix it here directly

File: ChemoTracker/components/Login/Login.js [hunk 5]
```diff
@@ -26,39 +23,45 @@ export default class Login extends React.Component {
   }
 
   async onSubmit() {
-    this.setState({ errorLabel: '' });
-    if (this.state.loginUsername === '') {
-      this.setState({ errorLabel: 'Username is required' });
+    const api = 'http://ec2-52-15-106-40.us-east-2.compute.amazonaws.com:8000';
```
Comment (jpatricia) [inline ChemoTracker/components/Login/Login.js]: you can make the api call in services folder as what i made for symptom history :) 
i have the api constant there and the login api call

Comment (shayywun) [reply ChemoTracker/components/Login/Login.js]: i need this to switch to next screen on success though

Comment (jpatricia) [reply ChemoTracker/components/Login/Login.js]: please see Calendar.js, you can see i imported the function there(line 12), and call the function with .then on line 56 of Calendar.js

File: ChemoTracker/components/Login/Login.js (hunk 2 above)
Comment (shayywun) [reply ChemoTracker/components/Login/Login.js]: why is it necessary? i dont need anything from the parent

Comment (lsivasothy) [reply ChemoTracker/components/Login/Login.js]: It doesn't really matter I guess

File: ChemoTracker/components/Login/Login.js (hunk 5 above)
Comment (jpatricia) [reply ChemoTracker/components/Login/Login.js]: line 37-50 can be used as example too

File: ChemoTracker/components/Login/Login.js [hunk 6]
```diff
@@ -26,39 +23,45 @@ export default class Login extends React.Component {
...
+    if (this.state.loginUsername === ''){
+      this.setState({errorLabel:'Username is required'});
       return;
     }
     if (this.state.loginPassword === '') {
       this.setState({ errorLabel: 'Password is required' });
       return;
     }
-    const response = fetch(`${api}/rest-auth/login/`, {
+    var response = fetch(`${api}/rest-auth/login/`, {
```
Comment (jpatricia) [inline ChemoTracker/components/Login/Login.js]: use `let` instead of `var`

File: ChemoTracker/components/Login/Login.js (hunk 5 above)
Comment (shayywun) [reply ChemoTracker/components/Login/Login.js]: okies i'll refactor it

Comment (shayywun) [reply ChemoTracker/components/Login/Login.js]: can't get it to work. it gives me undefined even though it logs in

**Review by jpatricia — APPROVED** [review]

**Pull request merged**